core/src/main/java/org/opentosca/toscana/core/data/LogEntry.java

```diff
@@ -0,0 +1,16 @@
+package org.opentosca.toscana.core.data;
+
+public class LogEntry {
+
+    public final long TIMESTAMP;
+    public final String MESSAGE;
+    public final LogLevel LOG_LEVEL;
+
+    LogEntry(long timestamp, String message, LogLevel level){
+        assert message != null;
```

Comment:
Don't do this!
If the condition is false, the whole application will crash because it throws a AssertionError which is not handled anywhere (at least usually)

I would replace it with a IllegalArgumentException

Reply:
Your right, will replace asserts with runtime exceptions